.gitignore

```diff
@@ -0,0 +1,40 @@
+.idea/
+vendor
+*.exe
+glide.lock
+*.DS_Store
+docker/docker-compose.yml
+.project
+.vscode/
+go.sum
+coverage.out
+debug
+*.dll
+
+# binary files
+cmd/config-seed/config-seed
+cmd/core-command/core-command
+cmd/core-data/core-data
+cmd/core-metadata/core-metadata
+cmd/export-client/export-client
+cmd/export-distro/export-distro
+cmd/support-logging/support-logging
+cmd/support-notifications/support-notifications
+cmd/sys-mgmt-agent/sys-mgmt-agent
+cmd/support-scheduler/support-scheduler
+cmd/sys-mgmt-executor/sys-mgmt-executor
+docs/_build/
```

Comment:
Recommend this file only contain what is relevant to this project rather than copied from edgex-go. If it only contains the exclusions for this project, it will be easier to reconcile (add) when we determine its eventual project repo. Also the name of this executable isn't included in the binary files section.